### DISCUSSION TITLE
Recommended companion app on Android

### DISCUSSION THREAD

**Discussion by helmecke** [zk-org/zk#229] [Help]
Hi, is there any companion app on Android that you can recommend? I've tried a bunch but could not get any to work for basic features.

Necessary features:
- [ ] List notes with titles not filename
- [ ] Link navigation
- [ ] Create note with zk filename schema

Thank you in advance!

**Comment by mickael-menu** [comment]
Personally I'm not writing notes on my phone, though I use Todoist on Android to quickly capture ideas in an inbox.

I know that Obsidian has an Android app that might check most requirements: https://obsidian.md/mobile

**Comment by nmeum** [comment]
Also currently looking into this. I think using [markor](https://github.com/gsantner/markor) with [syncthing](https://github.com/syncthing/syncthing-android) might/should work to some degree, though of cause it does not know about the zk file naming scheme, so it's not perfect but it's a start.

**Comment by zalegrala** [comment]
I was using GitJournal, but they have a bug where only the `master` branch is supported.

**Comment by mosk-it** [comment]
Has anything changed in the last two years?

my WIP worflow includes
termux that works great with git and zk
and termux-widget that adds widgets (icons) to your own scripts

so together they can open `zk` created note in editor with one click, for example inbox style like
```sh
export ZK_NOTEBOOK_DIR="/storage/emulated/0/DCIM/Notes"
NOTEPATH="$(zk new --no-input --print-path)"

am start -a android.intent.action.EDIT \
         -t "text/markdown" \
         -d "file://$NOTEPATH"
```
it works with markor, some editors might require defining activity of the app (you can find that with `dumpsys`)


Obsidian already was mentioned and some people [1] that work with zk recommend it for tag searching and editing

Do you use anything more suited?

[1] https://charnley.github.io/blog/2026/01/03/note-taking-for-programmers-zk-vscode-vim.html

**Comment by charnley** [comment]
I use Obsidian on my phone;

- to create daily todo, which is configured to same format as `zk`
- search in tags and content

I have not been able to configure it to create new random files in the same format as my zk-config though, but I rarely use it like that. Mobile is for quick note that I will then clean up later on my laptop.

It is relatively easy to configure Obsidian and GitSync to use the same repository as zk[1]. With GitHub/GitLab, but personally I use a VPS from a cloud provider.

[1] https://charnley.github.io/blog/2026/01/03/note-taking-for-programmers-zk-vscode-vim.html#mobile-compatible-setup
